lava/magma/compiler/channels/pypychannel.py

```diff
@@ -15,6 +15,8 @@
     AbstractCspSendPort,
     AbstractCspRecvPort,
 )
+from lava.magma.runtime.message_infrastructure.message_infrastructure_interface\
+    import MessageInfrastructureInterface
 
 
 @dataclass
```

Comment:
Please take a look at the ToDo below to not create first time attributes outside of __init__(..)
-> _array, _semaphore, _queue